src/site/content/en/mini-apps/mini-app-devtools/index.md

```diff
@@ -0,0 +1,242 @@
+- [Quick App demo](https://github.com/quickappcn/sample)
+
+## Development flow
+
+After launching the IDE and loading or creating a (demo) mini app, the first step is always to log
+in. Usually you just need to scan a QR code with the super app (where you are already logged in)
+that is generated by the IDE. Very rarely do you have to enter a password. Once you are logged in,
+the IDE knows your identity and lets you start programming, debugging, testing, and submitting your
+app for review. In the following, you can see screenshots of the five IDEs mentioned in the
+paragraph above.
+
+<figure class="w-figure">
+  <img src="wechat-devtools.png" alt="WeChat DevTools application window showing simulator, code editor, and debugger.">
+  <figcaption class="w-figure">
+    WeChat DevTools with simulator, code editor, and debugger.
+  </figcaption>
+</figure>
+
+<figure class="w-figure">
+  <img src="alipay-devtools.png" alt="Alipay DevTools application window showing code editor, simulator, and debugger.">
+  <figcaption class="w-figure">
+    Alipay DevTools with code editor, simulator, and debugger.
+  </figcaption>
+</figure>
+
+<figure class="w-figure">
+  <img src="baidu-devtools.png" alt="Baidu DevTools application window showing simulator, code editor, and debugger.">
+  <figcaption class="w-figure">
+    Baidu DevTools with simulator, code editor, and debugger.
+  </figcaption>
+</figure>
+
+<figure class="w-figure">
+  <img src="bytedance-devtools.png" alt="ByteDance DevTools application window showing simulator, code editor, and debugger.">
+  <figcaption class="w-figure">
+    ByteDance DevTools with simulator, code editor, and debugger.
+  </figcaption>
+</figure>
+
+<figure class="w-figure">
+  <img src="quick-app-devtools.png" alt="Quick App DevTools application window showing code editor, simulator, and debugger.">
+  <figcaption class="w-figure">
+    Quick App DevTools with code editor, simulator, and debugger.
+  </figcaption>
+</figure>
+
+As you can see, the fundamental components of all IDEs are very similar. You always have a code
+editor based on the [Monaco Editor](https://microsoft.github.io/monaco-editor/), the same project
+that also powers [VS Code](https://github.com/Microsoft/vscode). In all IDEs, there is a debugger
+based on the [Chrome DevTools front-end](https://github.com/ChromeDevTools/devtools-frontend) with
+some modifications, more on those later (see [Debugger](/mini-app-devtools/#debugger)). The IDEs _per se_ are
+implemented either as [NW.js](https://nwjs.io/) or as [Electron](https://www.electronjs.org/) apps,
+the simulators in the IDEs are realized as an
+[NW.js `<webview>`](https://docs.nwjs.io/en/latest/References/webview%20Tag/) or
+[Electron `<webview>`](https://www.electronjs.org/docs/api/webview-tag), which in turn are based on
+a [Chromium `<webview>`](https://www.electronjs.org/docs/api/webview-tag). If you are interested in
```

Comment:
```suggestion
[Electron `<webview>` tag](https://www.electronjs.org/docs/api/webview-tag), which in turn are based on
a [Chromium `<webview>` tag](https://www.electronjs.org/docs/api/webview-tag). If you are interested in
```

Reply:
@tomayac This is probably not obvious to people who aren't Electron-literate. Can you please make this change throughout.